validation-test/stdlib/Hashing.swift

```diff
@@ -128,11 +128,30 @@ HashingTestSuite.test("_squeezeHashValue/UInt") {
   checkRange((UInt.max-10)..<(UInt.max-1))
 }
 
+HashingTestSuite.test("String/hashValue/topBitsSet") {
+#if _runtime(_ObjC)
+#if arch(x86_64) || arch(arm64)
+  // Make sure that we don't accidently throw away bits by storing the result of
+  // NSString.hash into an int in the runtime.
+
+  // This is the bit pattern that we xor to NSString's hash value.
+  let hashOffset = UInt(bitPattern: 0x429b_1266_0000_0000)
+  let hash = "efghijkl".hashValue
+  // When we are not equal to the top bit of the xor'ed hashOffset pattern
+  // there where some bits set.
+  let  topHashBits = UInt(bitPattern:hash) & 0xffff_ffff_0000_0000
```

Comment:
Missed the space after the colon here. Also whitespace error on the same line. Will fix.